index.html

```diff
@@ -22,7 +22,7 @@ <h1>Today's To Do</h1>
         </form>
         <hr>
         <ul id="to-do-list">
-            <!--TASKS ELEMENTS DINAMICALLY INSERTED WITH JAVASCRIPT-->
+            <!-- TASKS ELEMENTS DINAMICALLY INSERTED WITH JAVASCRIPT -->
```

Comment:
- I suggest you remove commented lines because commented codes makes your codebase look unclean